### PR TITLE
Results screen

### PR DESCRIPTION
**Pull request by kikettas**
Added results screen where user will be able to confirm/modify the analysis results. 

### How to test
Launch the app and press Start analysis to see the results screen.

### Merging
Automatic.

### Ticket 
Issue #7

### CODE REVIEW

File: GiniVisionExample/GiniVisionExample/AppCoordinator.swift [hunk 1]
```diff
@@ -34,18 +38,39 @@ final class AppCoordinator: Coordinator {
...
 
 // MARK: MainViewControllerDelegate
 
 extension AppCoordinator: MainViewControllerDelegate {
     
     func main(viewController: MainViewController, didTapStartAnalysis: ()) {
-        
+        let ibanExtraction = GINIExtraction(name: "iban", value: "DE 1234 5678 9123 4567", entity: "entity", box: [:])
+        let paymentRecipientExtraction = GINIExtraction(name: "paymentRecipient",
+                                                        value: "Rick Sanchez", entity: "entity", box: [:])
```
Comment (nsobadjiev) [inline GiniVisionExample/GiniVisionExample/AppCoordinator.swift]: Nice that Rick is using the app, but why do you have his hardcoded data into the production code?

PICKLE RICK!

**Review by nsobadjiev — CHANGES_REQUESTED** [review]

File: GiniVisionExample/GiniVisionExample/AppCoordinator.swift (hunk 1 above)
Comment (kikettas) [reply GiniVisionExample/GiniVisionExample/AppCoordinator.swift]: Just for testing the UI and give and example to Ben (he will give a better design to these screens). This and the rest of hardcode data will be removed in a future PR.

Comment (nsobadjiev) [reply GiniVisionExample/GiniVisionExample/AppCoordinator.swift]: Well, just make some screenshots for Ben then?

Comment (kikettas) [reply GiniVisionExample/GiniVisionExample/AppCoordinator.swift]: I uploaded a version to Hockey so he could test in his phone. As I said, this hardcoded data will be remove in upcoming PRs.

**Review by nsobadjiev — APPROVED** [review]

**Pull request merged**